.env.development

```diff
@@ -0,0 +1,5 @@
+REACT_APP_OIDC_AUTHORITY="http://tunnistamo-backend:8000/"
+REACT_APP_PROFILE_API="http://tunnistamo-backend:8080/v1"
+REACT_APP_OIDC_API_TOKEN="http://tunnistamo-backend:8000/api-tokens/"
+REACT_APP_OIDC_REDIRECT_URI="http://tunnistamo-backend:3000/callback"
+REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/kukkuu"
```

Comment:
Not sure if it's ok to push these publically.

Reply:
Hm, I wonder why? 
They don't contain any secrets..

Reply:
@Nizar-Rahme this is just for development-environment. Besides, there can't be anything secret in JS-bundles since it's all visible in the browser. So CRA-env is not actually secrets, think of them as configuration for different environments.

Reply:
Right!